core/services/chainlink/application.go

```diff
@@ -399,7 +400,6 @@ func NewApplication(cfg *config.Config, ethClient eth.Client, advisoryLocker pos
 			return nil
 		}})
 	}
-	app.HeadTracker = headTracker
```

Comment:
Do we need to remove this field from the application struct

Reply:
oh, i just moved this assignment to struct creation